challenge/src/components/Pages/Home.jsx

```diff
@@ -5,6 +5,7 @@ import strings from "../../strings.json";
 import Fade from "react-reveal/Fade";
 import Table from '../Table'
 import GIF from "../Gif"
+import CurrentMonthAndYear from '../SomeInformationForTimeTravelers';
```

Comment:
the convention usually is the component should probably be the same name as the file, but it is also named something else in the other file. make it so everything has the same name `CurrentMonthAndYear`

Reply:
done